db/coordination.go

```diff
@@ -77,3 +77,32 @@ func (b *BothDatabases) UpdateUserDataUsage(context context.Context, username st
 
 	return res, err
 }
+
+func (b *BothDatabases) UpdateUserDataUsageBatch(context context.Context, start, end string) error {
+	// should pass in qualified usernames, icatdb method will strip it as needed
+	usages, err := b.icatdb.BatchCurrentDataUsage(context, start, end)
+	if err != nil {
+		return err
+	}
+
+	log.Tracef("usages in batch: %+v", usages)
+
+	var us []string
+	usagesFixed := make(map[string]int64)
+	for usr, usg := range usages { // keys of usages map
+		us = append(us, b.icatdb.FixUsername(usr))
+		usagesFixed[b.icatdb.FixUsername(usr)] = usg
+	}
```

Comment:
This is the perpetual issue that our DB wants to have usernames with suffixes but the ICAT doesn't. I might try to streamline it more at some point, but...

Reply:
We should probably remove the suffixes when we get some extra time for refactoring. The username suffix doesn't provide anything useful yet, and chances are that it never will.

Reply:
Yeah, true. I actually found the other day that we have some users in our DB that don't have that suffix, because they have what appear to just be their email addresses. I have no idea how or when that happened, but we'd undoubtedly need to figure out a way to clean those up at the same time.